server/bomberbloke_server.cpp

```diff
@@ -7,23 +7,29 @@
 #include <cereal/archives/portable_binary.hpp>
 #include <memory>
 #include <random>
+#include <stdio.h>
 
 int _default_bomb_time = DEFAULT_BOMB_TIMER;
 double _bloke_size[2] = { DEFAULT_BLOKE_SIZE, DEFAULT_BLOKE_SIZE };
 std::vector<int> _spawn_points = { 5, 5 };
 int colours[50][3];
 
 int
-main()
+main(int argc, char **argv)
 {
+  bool debug = false;
+  if(argc > 1 && strcmp(argv[1], "-debug") == 0) {
```

Comment:
By bash convention verbose arguments use a double dash, where single letters use a single dash

Reply:
We can definitely do better in terms of command line arguments - I've opened #91 to work on that another time